test/e2e/service_to_service_test.go

```diff
@@ -210,6 +210,8 @@ func TestServiceToServiceCall(t *testing.T) {
 	for _, tc := range testCases {
 		helloworldDomain := strings.TrimSuffix(resources.Route.Status.URL.Host, tc.suffix)
 		t.Run(tc.name, func(t *testing.T) {
+			cancel := logstream.Start(t)
```

Comment:
should we remove the one on top?

Reply:
no because there's a mix of resources inside and out 🙄